LeagueOfTowers/Assets/Resources/Map.txt

```diff
@@ -1,10 +1,10 @@
-00000000000000000-
-11110011111100000-
-11110011111100000-
-00110011001100000-
-00110011001100000-
-00110011001100000-
-00110011001100000-
-00111111001111000-
-00111111001111000-
-00000000000000000
+000000000000000000-
```

Comment:
So, do we need the Map.txt and MapSlim.txt if they are the same? I think it would be better to delete the MapSlim.txt

Reply:
I will delete the MapSlim.txt